### PR TITLE
First draft of my first linode guide

### PR DESCRIPTION
**Pull request by cjungmann**
This guide shows how to configure Apache to use a CGI/FastCGI script/program as a file handler.

### CODE REVIEW

**Comment by alexfornuto** [comment]
@cjungmann I suggest you edit your PR name to be the name of your guide, "Configuring a Script Intepreter for Apache"

File: docs/websites/apache-tips-and-tricks/apache-new-file-handler.md [hunk 1]
```diff
@@ -0,0 +1,589 @@
+---
+author:
+  name: Chuck Jungmann
+  email: chuck@cpjj.net
+description: 'Using a CGI or FastCGI Script as a File Handler'
+keywords: 'apache,cgi,fastcgi,AddHandler,SetHandler,AddType,mod_fastcgi'
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
```
Comment (alexfornuto) [inline docs/websites/apache-tips-and-tricks/apache-new-file-handler.md]: Linode uses the 4.0 version of the CC license.

File: docs/websites/apache-tips-and-tricks/apache-new-file-handler.md [hunk 2]
```diff
@@ -0,0 +1,589 @@
...
+title: 'Configuring a Script Intepreter for Apache'
+external_resources:
+ - '[Apache HTTP Server Version 2.4 Documentation](https://httpd.apache.org/docs/2.4/)'
+ - '[Module mod_fastcgi](https://htmlpreview.github.io/?https://github.com/FastCGI-Backups/mod_fastcgi/blob/master/docs/mod_fastcgi.html)'
+ - '[Archive of FastCGI.com](https://github.com/FastCGI-Archives/FastCGI.com/blob/master/README.md)'
+ - '[CGI Environment Variables](http://cgi101.com/learn/ch3/text.html)'
+ - '[Debugging with GDB](http://www.delorie.com/gnu/docs/gdb/gdb_toc.html)'
+---
+
+I have written a script interpreter that I want Apache to use when
```
Comment (alexfornuto) [inline docs/websites/apache-tips-and-tricks/apache-new-file-handler.md]: They generally don't use first-person voice in their guides.

File: docs/websites/apache-tips-and-tricks/apache-new-file-handler.md [hunk 3]
```diff
@@ -0,0 +1,589 @@
...
+doing this online, despite how easy it turned out to be.  I have written
+this guide to help others succeed where I struggled to understand.
+
+This guide will use a simple CGI script to demonstrate how to configure
+Apache to use the script as a file handler when an appropriate file is
+called.  Then it will show how to convert the CGI script to a FastCGI
+script for performance reasons, and how to modify the configuration to
+use the FastCGI version instead of the CGI script.
+
+The main sections are arranged thus:
```
Comment (alexfornuto) [inline docs/websites/apache-tips-and-tricks/apache-new-file-handler.md]: Linode's docs build in a T.O.C. automatically, making this redundant.

File: docs/websites/apache-tips-and-tricks/apache-new-file-handler.md [hunk 4]
```diff
@@ -0,0 +1,589 @@
...
+- [Prerequisites](#Prerequisites)
+- [The CGI Script](#TheCGIScript)
+- [Configuring Apache to Use a File Handler](#ConfiguringApache)
+- [Test CGI File Handler](#TestCGIHandler)
+- [Upgrading CGI to FastCGI](#UpgradeToFastCGI)
+- [Test FastCGI File Handler](#TestFastCGIHandler)
+- [Reading the Target File](#ReadingTheTargetFile)
+- [CGI/FastCGI Programming Suggestions](#ScriptProgrammingSuggestions)
+
+## Prerequisites                            {#Prerequisites}
```
Comment (alexfornuto) [inline docs/websites/apache-tips-and-tricks/apache-new-file-handler.md]: You can also get rid of ` {#Prerequisites}` as anchors will be created automatically.

File: docs/websites/apache-tips-and-tricks/apache-new-file-handler.md [hunk 5]
```diff
@@ -0,0 +1,589 @@
...
+       sudo /etc/init.d/apache2 restart
+
+3. Install `build-essential` tools for for compiling the C-language example CGI
+   and FastCGI programs and the FastCGI libraries.
+   
+       sudo apt-get install build-essential
+
+### FastCGI Setup                           {#FastCGISetup}
+
+4. Download, compile, and install FastCGI libraries.
```
Comment (alexfornuto) [inline docs/websites/apache-tips-and-tricks/apache-new-file-handler.md]: Numbered lists shouldn't span across sub-sections. New list, new numbers.

File: docs/websites/apache-tips-and-tricks/apache-new-file-handler.md [hunk 6]
```diff
@@ -0,0 +1,589 @@
...
+
+Many languages can be used to program a CGI script.  The example in this
+guide is a C program because I prefer that language, but it could as easily
+be written in Perl, Python, Java, or BASH.
+
+The example CGI script does nothing except announce its presence.
+
+1. Put the following source code into a file for compilation:
+
+   {:file} simplescript.c
```
Comment (alexfornuto) [inline docs/websites/apache-tips-and-tricks/apache-new-file-handler.md]: Incorrect callout. Check the [formatting guide](https://www.linode.com/docs/linode-writers-formatting-guide#files-and-file-excerpts)

**Comment by pbzona** [comment]
@cjungmann Thanks for submitting this. While it's a great topic, we typically try to steer clear of guides that involve a significant amount of custom code or development work. We're going to be unable to publish this guide for that reason, but we'd love to consider working with you on a future project. I believe I have your email, I'll reach out soon with information.

**Pull request closed without merging**